angular/src/components/lock.component.ts

```diff
@@ -60,6 +61,7 @@ export class LockComponent implements OnInit {
             (await this.cryptoService.hasKeyStored('biometric') || !this.platformUtilsService.supportsSecureStorage());
         this.biometricText = await this.storageService.get(ConstantsService.biometricText);
         this.email = await this.userService.getEmail();
+        this.hideInput = await this.keyConnectorService.getUsesKeyConnector() && !this.pinLock;
 
         // Users with key connector and without biometric or pin has no MP to unlock using
         if (await this.keyConnectorService.getUsesKeyConnector() && !(this.biometricLock || this.pinLock)) {
```

Comment:
Should we only call `await this.keyConnectorService.getUsesKeyConnector()` once and put the result into a variable. Saves us waiting for another async/await call.

Reply:
Maybe.